tests/lib/linter/Linter.test.js

```diff
@@ -0,0 +1,184 @@
+const { Linter } = require("../../../lib/linter");
+const { Rules } = require("../../../lib/rules");
+
+const config = {
+    rules: {
+        indentation: "error",
+        no_repetitive_step_keyword: "error",
+    },
+};
+
+const mockProblem = [
+    {
+        ruleId: "indentation",
+        type: "error",
+        message: "Indentation error",
+        location: {},
+    },
+];
+
+describe("class: Linter", () => {
+    afterEach(() => {
+        jest.clearAllMocks();
+        jest.restoreAllMocks();
```

Comment:
```suggestion
        jest.clearAllMocks();
        jest.restoreAllMocks();
```
Do we need both of them? or only one is sufficient?

Reply:
I overused it :sweat_smile: